app/server.js

```diff
@@ -27,6 +29,18 @@ export async function createHttpServer() {
   app.use(compression())
   app.use(bodyParser.json())
 
+  client.register.clear()
+  app.use(
+    promBundle({
+      includePath: true,
+      promClient: {
+        collectDefaultMetrics: {
+          prefix: 'dscp_identity_service',
```

Comment:
you need underscore at the end otherwise it will look like `your_namesomemetricname`